snippets/json-schema.code-snippets

```diff
@@ -1,118 +1,118 @@
 {
-	"Number property": {
+	"Snowboard: Number property": {
         "prefix": ["number"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": \"number\"", "}"],
-        "description": "A for loop."
+        "body": ["\"${1:property_name}\": {", "\t\"type\": \"number\",", "\t\"description\": \"${2:Description goes here}\"", "}${0}"],
+        "description": "A property of type 'number'."
     },
-    "String property": {
+    "Snowboard: String property": {
         "prefix": ["string"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": \"string\"", "}"],
-        "description": "A for loop."
+        "body": ["\"${1:property_name}\": {", "\t\"type\": \"string\",", "\t\"description\": \"${2:Description goes here}\"", "}${0}"],
+        "description": "A property of type 'string'."
     },
-    "Integer property": {
-        "prefix": ["string"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": \"integer\"", "}"],
-        "description": "A for loop."
+    "Snowboard: Integer property": {
+        "prefix": ["integer"],
+        "body": ["\"${1:property_name}\": {", "\t\"type\": \"integer\",", "\t\"description\": \"${2:Description goes here}\"", "}${0}"],
+        "description": "A property of type 'integer'."
     },
-    "Boolean property": {
+    "Snowboard: Boolean property": {
         "prefix": ["boolean"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": \"boolean\"", "}"],
+        "body": ["\"${1:property_name}\": {", "\t\"type\": \"boolean\",", "\t\"description\": \"${2:Description goes here}\"", "}${0}"],
         "description": "A for loop."
     },
```

Comment:
Maybe we should also have a generic "Add property" that lets you pick the type from a [choice](https://code.visualstudio.com/docs/editor/userdefinedsnippets#_choice)?

Reply:
Added a generic add property field. But I have kept it for only basic types for now.